src/librbd/io/ReadResult.cc

```diff
@@ -153,14 +153,16 @@ void ReadResult::C_ImageReadRequest::finish(int r) {
                  << dendl;
   if (r >= 0) {
     size_t length = 0;
+    std::vector<pair<uint64_t,uint64_t>> buffer_extents;
```

Comment:
I knew this issue sounded familiar ... it turns out I had already fixed it here [1] in one of my WIP branches.

[1] https://github.com/dillaman/ceph/blob/wip-librbd-migration-1/src/librbd/io/ReadResult.cc#L155:L160

Reply:
Yes, exactly same.